package.json

```diff
@@ -95,12 +97,17 @@
     "test-electron": "browserify dist/test/electron.js | testron | faucet",
     "start": "budo example/main.js:example/bundle.js --live --open --port 3000 --css example/main.css --title 'deck.gl' --watch-glob '**/*.{html,css,js,glsl}' -- -t babelify -t envify"
   },
-  "standard": {
-    "ignore": [
-      "**/build/**",
-      "**/dist/**",
-      "**/examples/**",
-      "**/Documents/**"
-    ]
+  "eslintConfig": {
```

Comment:
personally, I prefer to keep them in the dot files so that you can copy them over different projects.
I can easily forget I need to copy over part of the package.json file when scaffolding another project.